### ISSUE TITLE
[IPv6] unable to create VS using IPv6 range

### ISSUE THREAD

**Issue by tkam8** [F5Networks/f5-ipam-controller#31]
#### Setup Details
 FIC Version : 0.1.2
 CIS Version : 2.4.0 
 FIC Build: f5networks/f5-ipam-controller:latest
 CIS Build: f5networks/k8s-bigip-ctlr:latest       
 BIGIP Version: Big IP 15.1.2.1    
 AS3 Version: 3.26      
 Orchestration: K8S     
 Orchestration Version:   1.20        
 Additional Setup details: Calico, NodePort

#### Description  

Unable to create virtualService when using IPv6 range. 

#### Steps To Reproduce
1) Create range using IPv6 addresses
2) Deploy VS referencing the range
3) Confirm error

#### Expected Result 

IPv6 address allocated from ipv6 range

#### Actual Result  

Error 

#### Diagnostic Information
```
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    name: f5-ipam-controller
  name: f5-ipam-controller
  namespace: kube-system
spec:
  replicas: 1
  selector:
    matchLabels:
      app: f5-ipam-controller
  template:
    metadata:
      labels:
        app: f5-ipam-controller
    spec:
      containers:
      - args:
        - --orchestration=kubernetes
        - --ip-range='{"Test":"10.192.75.113-10.192.75.116","Production":"10.192.125.30-10.192.125.50","ipv6":"240b:c0e0:105:2870:6518:0002:1:0001-240b:c0e0:105:2870:6518:0002:1:0020"}'
        - --log-level=DEBUG
        command:
        - /app/bin/f5-ipam-controller
        image:  f5networks/f5-ipam-controller:0.1.2
        imagePullPolicy: IfNotPresent
        name: f5-ipam-controller
      serviceAccount: ipam-ctlr
      serviceAccountName: ipam-ctlr
```  

```
apiVersion: "cis.f5.com/v1"
kind: VirtualServer
metadata:
  name: f5-demo-mysite
  labels:
    f5cr: "true"
spec:
  host: mysite.f5demo.com
  ipamLabel: ipv6
  pools:
  - monitor:
      interval: 20
      recv: ""
      send: /
      timeout: 31
      type: http
    path: /
    service: f5-demo
    servicePort: 80
```

FIC error:
```
# kubectl logs f5-ipam-controller-6c9bcdcd8d-rxpqx -n kube-system
2021/05/16 13:55:26 [DEBUG] Creating IPAM Kubernetes Client
2021/05/16 13:55:26 [INFO] [INIT] Starting: F5 IPAM Controller - Version: 0.1.2, BuildInfo: azure-215-77b1d12be7e9c4c344e1747451d18be704a3b30b
2021/05/16 13:55:26 [DEBUG] [ipam] Creating Informers for Namespace kube-system
2021/05/16 13:55:26 [DEBUG] Created New IPAM Client
2021/05/16 13:55:26 [DEBUG] [MGR] Creating Manager with Provider: f5-ip-provider
2021/05/16 13:55:26 [DEBUG] [STORE] [id ipaddress status ipam_label]
2021/05/16 13:55:26 [DEBUG] [STORE]  1	 10.192.75.113 1 Test
2021/05/16 13:55:26 [DEBUG] [STORE]  2	 10.192.75.114 1 Test
2021/05/16 13:55:26 [DEBUG] [STORE]  3	 10.192.75.115 1 Test
2021/05/16 13:55:26 [DEBUG] [STORE]  4	 10.192.125.30 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  5	 10.192.125.31 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  6	 10.192.125.32 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  7	 10.192.125.33 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  8	 10.192.125.34 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  9	 10.192.125.35 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  10	 10.192.125.36 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  11	 10.192.125.37 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  12	 10.192.125.38 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  13	 10.192.125.39 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  14	 10.192.125.40 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  15	 10.192.125.41 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  16	 10.192.125.42 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  17	 10.192.125.43 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  18	 10.192.125.44 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  19	 10.192.125.45 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  20	 10.192.125.46 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  21	 10.192.125.47 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  22	 10.192.125.48 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  23	 10.192.125.49 1 Production
2021/05/16 13:55:26 [DEBUG] [STORE]  24	 240b:c0e0:105:2870:6518:2:1:1 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  25	 240b:c0e0:105:2870:6518:2:1:2 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  26	 240b:c0e0:105:2870:6518:2:1:3 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  27	 240b:c0e0:105:2870:6518:2:1:4 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  28	 240b:c0e0:105:2870:6518:2:1:5 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  29	 240b:c0e0:105:2870:6518:2:1:6 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  30	 240b:c0e0:105:2870:6518:2:1:7 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  31	 240b:c0e0:105:2870:6518:2:1:8 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  32	 240b:c0e0:105:2870:6518:2:1:9 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  33	 240b:c0e0:105:2870:6518:2:1:a 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  34	 240b:c0e0:105:2870:6518:2:1:b 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  35	 240b:c0e0:105:2870:6518:2:1:c 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  36	 240b:c0e0:105:2870:6518:2:1:d 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  37	 240b:c0e0:105:2870:6518:2:1:e 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  38	 240b:c0e0:105:2870:6518:2:1:f 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  39	 240b:c0e0:105:2870:6518:2:1:10 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  40	 240b:c0e0:105:2870:6518:2:1:11 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  41	 240b:c0e0:105:2870:6518:2:1:12 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  42	 240b:c0e0:105:2870:6518:2:1:13 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  43	 240b:c0e0:105:2870:6518:2:1:14 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  44	 240b:c0e0:105:2870:6518:2:1:15 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  45	 240b:c0e0:105:2870:6518:2:1:16 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  46	 240b:c0e0:105:2870:6518:2:1:17 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  47	 240b:c0e0:105:2870:6518:2:1:18 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  48	 240b:c0e0:105:2870:6518:2:1:19 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  49	 240b:c0e0:105:2870:6518:2:1:1a 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  50	 240b:c0e0:105:2870:6518:2:1:1b 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  51	 240b:c0e0:105:2870:6518:2:1:1c 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  52	 240b:c0e0:105:2870:6518:2:1:1d 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  53	 240b:c0e0:105:2870:6518:2:1:1e 1 ipv6
2021/05/16 13:55:26 [DEBUG] [STORE]  54	 240b:c0e0:105:2870:6518:2:1:1f 1 ipv6
2021/05/16 13:55:26 [INFO] [CORE] Controller started
2021/05/16 13:55:26 [INFO] Starting IPAMClient Informer
I0516 13:55:26.362609       1 shared_informer.go:240] Waiting for caches to sync for F5 IPAMClient Controller
2021/05/16 13:55:26 [DEBUG] Enqueueing on Create: kube-system/ipam.172.28.15.147.k8s
2021/05/16 13:55:26 [DEBUG] Enqueueing on Create: kube-system/ipam.k8s
I0516 13:55:26.463485       1 shared_informer.go:247] Caches are synced for F5 IPAMClient Controller
2021/05/16 13:55:26 [DEBUG] K8S Orchestrator Started
2021/05/16 13:55:26 [DEBUG] Starting Response Worker
2021/05/16 13:55:26 [DEBUG] Starting Custom Resource Worker
2021/05/16 13:55:26 [DEBUG] Processing Key: &{0xc00016e000 <nil> Create}
2021/05/16 13:55:26 [DEBUG] Processing Key: &{0xc00016e148 <nil> Create}
2021/05/16 13:55:26 [DEBUG] [CORE] Allocated IP: 240b:c0e0:105:2870:6518:2:1:1 for Request:
Hostname: mysite.f5demo.com	Key: 	CIDR: 	IPAMLabel: ipv6	IPAddr: 	Operation: Create

2021/05/16 13:55:26 [ERROR] [IPMG] Unable to Create 'A' Record, as Invalid IP Address Provided
2021/05/16 13:55:26 [ERROR] Unable to Update F5IPAM: kube-system/ipam.172.28.15.147.k8s	 Error: F5IPAM.fic.f5.com "ipam.172.28.15.147.k8s" is invalid: status.IPStatus.ip: Invalid value: "": status.IPStatus.ip in body should match '^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$'
2021/05/16 13:55:26 [DEBUG] Updated: kube-system/ipam.172.28.15.147.k8s with Status. With IP: 240b:c0e0:105:2870:6518:2:1:1 for Request:
Hostname: mysite.f5demo.com	Key: 	CIDR: 	IPAMLabel: ipv6	IPAddr: 240b:c0e0:105:2870:6518:2:1:1	Operation: Create
```

#### Observations (if any)

**Comment by trinaths** [comment]
Created CONTCNTR-2383 for internal tracking

**Comment by sravyap135** [comment]
IPv6 support will be available from FIC 0.1.6, CIS 2.7.0 release
